src/main/java/org/prebid/server/bidder/adform/AdformBidder.java

```diff
@@ -71,6 +73,15 @@ public AdformBidder(String endpointUrl) {
             return Result.of(Collections.emptyList(), errors);
         }
 
+        String currency = DEFAULT_CURRENCY;
+        final List<String> currencies = request.getCur();
+        if (CollectionUtils.isNotEmpty(currencies)) {
+            final boolean hasUsd = currencies.stream().anyMatch(cur -> cur.equals(DEFAULT_CURRENCY));
```

Comment:
1. Hey, stream-guy :) i think we can use pretty old `contains(...)` method.
2. Pls extract determining currency to a separate method.